target/amavis/postfix-amavis.cf

```diff
@@ -23,7 +23,6 @@ smtp-amavis     unix    -       -       n       -       2       smtp
   -o smtpd_helo_restrictions=
   -o smtpd_sender_restrictions=
   -o smtpd_recipient_restrictions=permit_mynetworks,reject
-  -o smtpd_data_restrictions=reject_unauth_pipelining
```

Comment:
This restriction should remain here explicitly, as the issue explains.

It's not necessary, but easier to maintain if we don't have to think about what the implicit config inherited here is, especially when we expect it to be this.

If the change to skip restriction by trust does work, then all the more reason for leaving this as-is for Amavis probably? (_I haven't given much thought to the Amavis config we have here, I'd just rather not tamper with it_)

Reply:
Done